JumaKahiga/clean_arch
# Folder Structure

**What does this PR do?**
 - Sets up the overall folder structure
 - Creates a Django project, employees' app, and models.

**How should this be manually tested?**
- clone the repo
- check out to `ch-setting-up`
- cd into `clean_arch/infrastructure`
- Create a database and run `python manage.py migrate` to create relevant tables.
- To run the app run `python manage.py runserver`

**Screenshots**
![Screenshot 2019-08-28 at 11 00 22](https://user-images.githubusercontent.com/21296622/63836878-1a42e080-c983-11e9-88ba-a8dc43a6c266.png)
